pkg/secret/secret.go

```diff
@@ -343,6 +360,19 @@ var DefaultRules = map[string]Meta{
 			{Name: VaultToken, Required: true, Opaque: true, Description: "Token for Vault"},
 		},
 	},
+	SlackSecretType: {
+		Fields: []FieldMeta{
+			{Name: SlackApiUrl, Required: true, Opaque: true, Description: "Slack url to send alerts"},
+			{Name: SlackChannel, Required: true, Opaque: true, Description: "Channel name for the alerts"},
```

Comment:
Is this supposed to be in the secret? IMHO this is rather a feature setting. (Eg. use the same secret, but separate channels for separate clusters)